src/bin/casc/main.rs

```diff
@@ -76,9 +76,9 @@ fn main() -> std::io::Result<()> {
 fn print_error(error_list: CascadeErrors) -> std::io::Result<()> {
     for e in error_list {
         if let ErrorItem::Parse(p) = e {
-            p.print_diagnostic();
+            p.print_diagnostic(atty::is(atty::Stream::Stderr));
```

Comment:
I'd like to be able to keep colors even when not using a terminal e.g., piping into `less -R`. I then suggest to add a casc option `--color=WHEN` that defaults to `auto` (cf. `ls`or `grep` options).